de_build/adf/adf-pipeline.yml

```diff
@@ -27,6 +27,8 @@ pool:
 
 variables:
   - template: adf-pipeline-vars.yml
+  - template: air-data-testing-vars.yml
```

Comment:
What does air mean?

Reply:
Ah assuming it's Airflow. Need to check what we're using and remove unused vars. Also need to rename the file

Reply:
air mean ( amido independent runnner)